smart-contracts/Makefile

```diff
@@ -0,0 +1,33 @@
+# Makefile
+
+SHELL:=/bin/bash
+
+# Run slither over entire directory
+# `make slither`
+slither:
+	python3 -m venv venv
```

Comment:
Move these three lines to a new target?  We repeat them several times.

Reply:
Good point, updated!